core/trino-main/src/test/java/io/trino/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -245,6 +232,44 @@ else if (TINYINT.equals(type)) {
 
             setup(sliceOutput.slice(), pagesSerde, pages);
         }
+
+        private void writeValue(Type type, Object value, BlockBuilder blockBuilder)
```

Comment:
nit: you could make this a separate commit

Reply:
I mean extraction of `writeValue` could be a separate commit since it's pure refactor